docker/README.md

```diff
@@ -0,0 +1,21 @@
+# VDMS Docker
+
+The build command is: 
```

Comment:
Do we need to specify that this is for building the server only? I'd like to think people are smart enough to figure that out, but ... 

If we mention it somewhere else (like in the main README), that's also probably fine

Reply:
I added that to the README here, just in case.